sql/core/src/test/scala/org/apache/spark/sql/CollationRegexpExpressionsSuite.scala

```diff
@@ -149,38 +149,40 @@ class CollationRegexpExpressionsSuite extends QueryTest with SharedSparkSession
   test("Support StringSplit string expression with Collation") {
```

Comment:
try covering some more edge cases in these tests, such as: empty strings, uppercase and lowercase mix, different byte-length chars, etc.